### PR TITLE
add arity example

### PR DESCRIPTION
**Pull request by stoeffel**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -5,19 +5,35 @@
...
+```js
+const sum = (a, b) => a + b;
+
+const arity = sum.length;
+console.log(arity);
+// => 2
+// The arity of sum is 2
+```
+
+Helper modules in the wild.
```
Comment (hemanth) [inline readme.md]: This repo is not JS specific, so better remove this? 

Thank for the PR!

Comment (stoeffel) [inline readme.md]: > Thank for the PR!

NP

Is the example okay in JS or do you want the example also be more generic?

Comment (hemanth) [inline readme.md]: We should make it generic, but what you be the generic syntax?

**Comment by stoeffel** [comment]
IMHO JS is generic enough :smile: (Everyone knows JS).
But else we should add samples in pseudo-code. Your call.

**Comment by hemanth** [comment]
Heh heh ok, let's stick with this as of now.

**Comment by stoeffel** [comment]
:balloon:

**Pull request merged**